src/init/omp.fish

```diff
@@ -1,11 +1,19 @@
 function fish_prompt
     set -l omp_duration "$CMD_DURATION$cmd_duration"
```

Comment:
That's milliseconds, right? So the chance of being off here and accidentally setting it to 0 by accident is minimal?

Reply:
milliseconds right, but the lowest value I got is 4 with an empty command but you never know :)